shared/chat/conversation/messages/cards/new-chat.tsx

```diff
@@ -3,15 +3,46 @@ import * as Kb from '../../../../common-adapters'
 import * as Styles from '../../../../styles'
 import openUrl from '../../../../util/open-url'
 
-type Props = {}
+type Props = {
+  self: boolean
+  openPrivateFolder: () => void
+}
 
-const NewCard = (_: Props) => (
-  <Kb.Box2 direction="horizontal" style={styles.container} alignItems="flex-start">
+type InnerProps = {
+  action: () => void
+  icon: Kb.IconType
+  imageLower?: boolean
+  label: string
+  tall?: boolean
+  text: string
+}
+
+const NewCard = (props: Props) => {
+  const iconEncrypted: Kb.IconType = 'icon-illustration-encrypted-116-96'
+  const iconSecure: Kb.IconType = 'icon-illustration-secure-116-96'
+  const innerProps = props.self ? {
+    action: props.openPrivateFolder,
+    icon: iconSecure,
+    imageLower: true,
+    label: 'Open your private folder',
+    tall: true,
+    text: 'Use this chat to store secure notes such as credit card numbers, passwords, or secret keys.',
+  } : {
+    action: () => openUrl('https://keybase.io/docs/chat/crypto'),
+    icon: iconEncrypted,
+    label: 'Read more',
+    text: 'This conversation is end-to-end encrypted.',
+  }
+  return (<NewCardInner {...innerProps} />)
+}
+
+const NewCardInner = (props: InnerProps) => (
```

Comment:
why have this additional component

Reply:
In case a third card comes along it'd be easier to add to than this alternative:
```
  const label = prop.self ? 'Open your private folder' : 'Read more'
  const action = props.self ? props.openPrivateFolder : () => openUrl('https://keybase.io/docs/chat/crypto')
  const icon = props.self ? iconSecure : iconEncrypted
  const text = props.self ? 'Use this chat to store secure notes such as credit card numbers, passwords, or secret keys.' : 'This conversation is end-to-end encrypted.'
```

All the [cards](https://app.zeplin.io/project/56b25ffc64fc91773a73aaf8/screen/5d8ea88c29a6946198d0593b) are similar now it might make sense to unify their backing component in the future.

Wdyt?